lib/commits.js

```diff
@@ -120,15 +120,15 @@ const findCommitsWithAssociatedPullRequests = async ({
     allCommits,
     includedIds = {}
 
+  const since = lastRelease ? lastRelease.created_at : config['commits-since']
+
```

Comment:
Like mentioned, I think the following would help people debugging their manually-crafted dates

```suggestion
  const since = lastRelease ? lastRelease.created_at : config['commits-since']
  
  const validationResult = Joi.date().iso().utc().validate(since);

  // The validation result contains either an error or the validated value
  if (validationResult.error) {
    the.correct.logging.module('Validation error:', validationResult.error.message);
    theCorrectExitMethod(1)
  }
```